src/advertising.ts

```diff
@@ -76,6 +76,14 @@ export interface AdItem {
    * The fallback ad sources need to have the same `AdSourceType` as the main ad source.
    */
   sources: AdSource[];
+
+  /**
+   * @platform Android
+   * The amount of seconds the ad manifest is loaded in advance;
+   * i.e as compared to when the ad break is scheduled for playback.
+   * Default value is 0.0
```

Comment:
Please match the documentation here: v

Like:
```suggestion
   * The amount of seconds the ad manifest is loaded in advance
   * compared to when the ad break is scheduled for playback.
   *
   * Default value is 0.0
   *
   * @platform Android
```